Resources/public/css/styles.css

```diff
@@ -216,6 +216,11 @@ body.fixed .content-header .navbar {
     padding: 15px;
 }
 
+.sidebar-menu li.keep-open > .treeview-menu {
+    display: block !important;
+    height: auto !important;
```

Comment:
jQuery slideUp/slideDown functions use inline style attribute to set height in animation loop and set display block/none. I think `!important` is a proper tool to hack some visual issues in this case. You can see more examples in style.css i.e. around select2. The other way is to open PR in AdminLTE. Add an option to filter out some menu elements from animation in js there, then back to this PR and use build in AdminLTE option without hacking.